hack/observability/opentelemetry/readme.md

```diff
@@ -0,0 +1,118 @@
+# OpenTelemetry Tracing
+
+This directory contains resources to help deploy [opentelemetry-collector][1] and to export and view
+[traces][2] in Azure Application Insights.
+
+## View traces in Application Insights
+
+When creating a development cluster with `make tilt-up`, OpenTelemetry tracing is automatically
+enabled. Traces are exported to the App Insights resource that you specify
+with `AZURE_INSTRUMENTATION_KEY`.
+
+If you don't have an App Insights resource, create one through the Azure Portal
+at https://ms.portal.azure.com/#create/Microsoft.AppInsights.
+
+To the upper right of the "Overview" page for your App Insights resource you will see an entry
+for "Instrumentation Key".
+
+![Instrumentation Key](assets/portal_instrumentation_key.png)
+
+Use this value for `AZURE_INSTRUMENTATION_KEY` in `tilt-settings.json`:
```

Comment:
is this a shared value? shouldn't it be the dev's App insights resource key like stated above?

Reply:
I guess that's what that sentence means, my first initial reading of the doc (skipping directly to the copy pastable content) read this as "Use this value" as in the value below, maybe clarify this should be the value mentioned above

Reply:
I updated this language to be more clear.